### DISCUSSION TITLE
index.html

### DISCUSSION THREAD

**Discussion by whitepoll** [telekom-security/tpotce#1590] [Q&A]
Hello everyone,
is there by any chance a way to customize the start page of Tpopt? the one I find on port 64297?
Thanks

**Comment by mgrabovac22** [comment]
Yes, you can customize the start page of Tpopt on port 64297. Here are the general steps:
1. Locate the Start Page Files: Find the directory where the start page HTML files are located. This is usually specified in the Tpopt configuration.

2. Edit the HTML/CSS: Modify the HTML and CSS files to customize the appearance and content of the start page as needed.

3. Restart Tpopt: After making changes, restart the Tpopt service to apply the updates.
